public/components/datasources/components/__tests__/data_connection.test.tsx

```diff
@@ -15,6 +15,15 @@ import { DataConnection } from '../manage/data_connection';
 import ReactDOM from 'react-dom';
 import { coreRefs } from '../../../../../public/framework/core_refs';
 
+jest.mock('../../../../plugin', () => ({
+  getRenderAccelerationDetailsFlyout: jest.fn(() =>
+    jest.fn().mockImplementation(() => console.log('Acceleration Details Flyout Rendered'))
```

Comment:
Was the console.log added was debugging?

Reply:
removed.